### PR TITLE
test: update playwright tests to support sha256 and sha512 algorithms

### PR DESCRIPTION
**Pull request by paulseto**
### What does this PR do?

Execute playwright tests with additional sha256 and sha512 algorithms.

### Closes Issue(s)

No issue.

### Motivation

Certain customers requires the use of only sha512 for checksums.

### More

- [ ] Added/updated documentation

### CODE REVIEW

**Comment by ffdixon** [comment]
We've got a contributor agreement in place.

File: bigbluebutton-tests/playwright/.env.template [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # meeting credentials
 BBB_URL=""                        # https://DOMAIN_NAME/bigbluebutton/api
 BBB_SECRET=""
+CHECKSUM=""                       # Force a specific checksum algorithm. Useful for when the secret length is not a sha algorithm length
```
Comment (antobinary) [inline bigbluebutton-tests/playwright/.env.template:4]:
```suggestion
CHECKSUM=""                       # (sha1 / sha256 / sha512) Force a specific checksum algorithm. Useful for when the secret length is not a sha algorithm length
```

Let's include some of the options

Comment (antonbsa) [inline bigbluebutton-tests/playwright/.env.template:4]: We could add the possible inputs:
```suggestion
CHECKSUM=""                       # (sha1 | sha256 | sha512) Force a specific checksum algorithm. Useful for when the secret length is not a sha algorithm length
```

File: bigbluebutton-tests/playwright/core/helpers.js [hunk 2]
```diff
@@ -11,6 +11,24 @@ const { format } = require('node:util');
 const chalk = require('chalk');
 const parameters = require('./parameters');
 
+function getChecksum(text, secret) {
+  let algorithm = (process.env.CHECKSUM || '').toLocaleLowerCase();
+  if (!['sha1', 'sha256', 'sha512'].includes(algorithm)) {
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/core/helpers.js:16]: is there any specific reason / case for using `toLocaleLoweCase()` instead of `toLowerCase()`?

Comment (paulseto) [reply bigbluebutton-tests/playwright/core/helpers.js:16]: It was a typo from auto complete. `toLowerCase()` is fine.

File: bigbluebutton-tests/playwright/.env.template (hunk 1 above)
Comment (paulseto) [reply bigbluebutton-tests/playwright/.env.template:4]: Good suggestion.

File: bigbluebutton-tests/playwright/core/helpers.js [hunk 3]
```diff
@@ -11,6 +11,24 @@ const { format } = require('node:util');
 const chalk = require('chalk');
 const parameters = require('./parameters');
 
+function getChecksum(text, secret) {
+  let algorithm = (process.env.CHECKSUM || '').toLocaleLowerCase();
```
Comment (antobinary) [inline bigbluebutton-tests/playwright/core/helpers.js:15]:
```suggestion
  let algorithm = (process.env.CHECKSUM || '').toLowerCase();
```

**Pull request merged**